### PR TITLE
Feature: Improve tags context menu

### LINKED ISSUE #11457: Improve the right-click context menu in the tags widget
### What feature or improvement do you think would benefit Files?

The right click context menu for tag items currently only has shell items, we should add Open With, Open in new tab/window/pane, etc. when applicable.

### Requirements

- Add more items to right-click context menu in tags widget

### Files Version

2.4.40.0

### Windows Version

Windows 11 21H2

### Comments

_No response_

### PR DESCRIPTION
**Pull request by hecksmosis**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #11457

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [x] Tested the changes for accessibility

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

File: src/Files.App/UserControls/Widgets/FileTagsWidget.xaml.cs [hunk 1]
```diff
@@ -45,13 +59,27 @@ public FileTagsWidgetViewModel ViewModel
...
 		{
 			InitializeComponent();
 
 			// Second function is layered on top to ensure that OpenPath function is late initialized and a null reference is not passed-in
 			// See FileTagItemViewModel._openAction for more information
 			ViewModel = new(x => OpenAction!(x));
+			OpenInNewTabCommand = new RelayCommand<WidgetCardItem>(OpenInNewTab);
+			OpenInNewWindowCommand = new RelayCommand<WidgetCardItem>(OpenInNewWindow);
+			OpenFileLocationCommand = new RelayCommand<WidgetCardItem>(OpenFileLocation);
+			OpenInNewPaneCommand = new RelayCommand<WidgetCardItem>(OpenInNewPane);
```
Comment (yaira2) [inline src/Files.App/UserControls/Widgets/FileTagsWidget.xaml.cs:74]: Should there be one for properties?

Comment (hecksmosis) [reply src/Files.App/UserControls/Widgets/FileTagsWidget.xaml.cs:74]: Yes, I forgot to copy it here

**Comment by yaira2** [comment]
Should this have "send to" and "open with" as well?

**Comment by hecksmosis** [comment]
I think we could do that, but shouldn't we do it in another PR, as it would need to be implemented for the rest of the widgets?

**Comment by yaira2** [comment]
The other widgets already have this, I think we're only missing it for tags.

**Comment by hecksmosis** [comment]
Yes, deleted comment bc dumb

**Comment by hecksmosis** [comment]
Done

File: src/Files.App/UserControls/InnerNavigationToolbar.xaml [hunk 2]
```diff
@@ -696,7 +696,7 @@
 									Padding="0"
 									HorizontalContentAlignment="Center"
 									VerticalContentAlignment="Center"
-									AutomationProperties.Name="{helpers:ResourceString Name=LargeIcons}"
+									AutomationProperties.Name="{helpers:ResourceString Name=NavToolbarLargeIcons/AutomationProperties/Name}"
```
Comment (yaira2) [inline src/Files.App/UserControls/InnerNavigationToolbar.xaml:699]: This change looks wrong 🤔

Comment (hecksmosis) [reply src/Files.App/UserControls/InnerNavigationToolbar.xaml:699]: I don't remember having done that, must have been a merge or something, I'll revert now

File: src/Files.App/UserControls/Widgets/FileTagsWidget.xaml.cs [hunk 3]
```diff
@@ -60,48 +110,135 @@ private async void FileTagItem_ItemClick(object sender, ItemClickEventArgs e)
...
-
-				var openedPopups = Microsoft.UI.Xaml.Media.VisualTreeHelper.GetOpenPopups(App.Window);
-				var secondaryMenu = openedPopups.FirstOrDefault(popup => popup.Name == "OverflowPopup");
-
-				var itemsControl = secondaryMenu?.Child.FindDescendant<ItemsControl>();
-				if (itemsControl is not null)
+				new ContextMenuFlyoutItemViewModel()
+				{
+					Text = "OpenItemsWithCaptionText".GetLocalizedResource(),
+					Glyph = "\uE17D",
```
Comment (yaira2) [inline src/Files.App/UserControls/Widgets/FileTagsWidget.xaml.cs:144]: Can you use the new opacity icon for "open with"?

File: src/Files.App/UserControls/Widgets/FileTagsWidget.xaml.cs [hunk 4]
```diff
@@ -60,48 +110,135 @@ private async void FileTagItem_ItemClick(object sender, ItemClickEventArgs e)
...
+				{
+					Text = "SendTo".GetLocalizedResource(),
+					Tag = "SendToPlaceholder",
+					IsEnabled = false,
+					ShowItem = !isFolder
+				},
+				new ContextMenuFlyoutItemViewModel()
+				{
+					Text = "SideBarOpenInNewTab/Text".GetLocalizedResource(),
+					Glyph = "\uF113",
```
Comment (yaira2) [inline src/Files.App/UserControls/Widgets/FileTagsWidget.xaml.cs:159]: Same for open in new tab

File: src/Files.App/UserControls/Widgets/FileTagsWidget.xaml.cs [hunk 5]
```diff
@@ -60,48 +110,135 @@ private async void FileTagItem_ItemClick(object sender, ItemClickEventArgs e)
...
+					Text = "OpenFileLocation".GetLocalizedResource(),
+					Glyph = "\uED25",
+					Command = OpenFileLocationCommand,
+					CommandParameter = item,
+					ShowItem = !isFolder
+				},
+				new ContextMenuFlyoutItemViewModel()
+				{
+					Text = "OpenInNewPane".GetLocalizedResource(),
+					ColoredIcon = new ColoredIconModel()
```
Comment (yaira2) [inline src/Files.App/UserControls/Widgets/FileTagsWidget.xaml.cs:184]: And open in new pane

**Review by yaira2 — DISMISSED** [review]

**Pull request merged**